specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -125,47 +116,161 @@
+            "description": "The service has accepted the request for processing.  The client can query the operation status and retrieve the analysis result using the 'Operation-Location' URL header specified in the response. Note that the 'Operation-Location' URL expires in 48 hours.",
             "headers": {
               "Operation-Location": {
-                "description": "URL to query for status of the operation. The URL will expire in 48 hours. ",
-                "type": "string"
+                "type": "string",
+                "description": "URL to query and retrieve the analysis result. This URL expires in 48 hours."
               }
             }
-          },
-          "default": {
-            "description": "Error response.",
-            "schema": {
-              "$ref": "FormRecognizerReceiptOcr.json#/definitions/ComputerVisionError"
-            }
           }
         },
         "x-ms-examples": {
-          "Successful 'Batch Read Receipt' request": {
-            "$ref": "./examples/SuccessfulBatchReadReceiptWithStream.json"
+          "Successful 'Analyze Receipt' request": {
+            "$ref": "./examples/SuccessfulAnalyzeReceiptWithStream.json"
           }
         }
       }
     }
   },
   "definitions": {
-    "readReceiptResult": {
+    "OperationStatus": {
+      "type": "string",
+      "description": "Analysis operation status: NotStarted, Running, Failed, Succeeded.",
+      "enum": [
+        "NotStarted",
+        "Running",
+        "Failed",
+        "Succeeded"
+      ],
+      "x-ms-enum": {
+        "name": "OperationStatusCodes",
+        "modelAsString": false
+      },
+      "x-nullable": false
+    },
+    "TextRecognitionResult": {
+      "description": "Text recognition result from a page in the input document.",
+      "type": "object",
+      "required": [
+        "lines"
+      ],
+      "properties": {
+        "page": {
+          "description": "The 1-based page number of the recognition result.",
+          "type": "integer"
+        },
+        "clockwiseOrientation": {
+          "description": "The orientation of the image in clockwise direction, measured in degrees between [0, 360).",
+          "type": "number"
+        },
+        "width": {
+          "description": "The width of the image/PDF in pixels/inches, respectively.",
+          "type": "number"
+        },
+        "height": {
+          "description": "The height of the image/PDF in pixels/inches, respectively.",
+          "type": "number"
+        },
+        "unit": {
+          "description": "The unit used by the width, height and boundingBox properties. For images, the unit is \"pixel\". For PDF, the unit is \"inch\".",
+          "type": "string",
+          "enum": [
+            "pixel",
+            "inch"
+          ],
+          "x-ms-enum": {
+            "name": "TextRecognitionResultDimensionUnit",
+            "modelAsString": false
```

Comment:
Explicitly marking something as `"modelAsString": false` is not required (it is the default). But please make sure that this is intentional from a versioning perspective.